Sources/CoreFoundation/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift open source project
+##
+## Copyright (c) 2024 Apple Inc. and the Swift project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of Swift project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+file(GLOB_RECURSE _CoreFoundationSources "*.c")
+add_library(CoreFoundation STATIC ${_CoreFoundationSources})
+
+target_include_directories(CoreFoundation
+    PUBLIC
+        include
+    PRIVATE
+        internalInclude)
+
+set(_CoreFoundation_build_flags ${_Foundation_common_build_flags})
+list(APPEND _CoreFoundation_build_flags
+    "-DCF_CHARACTERSET_UNICODE_DATA_L=\"${CMAKE_SOURCE_DIR}/Sources/CoreFoundation/CFUnicodeData-L.mapping\""
```

Comment:
Do we need this anymore? I think I removed the need for it with a9ffa779b1?

Reply:
Ahh I must have missed that. I'll remove it.